Gemfile

```diff
@@ -16,8 +16,6 @@ gem "react-rails"
 
 gem 'accountingjs-rails'
 gem "jquery-rails"
-gem "turbolinks"
```

Comment:
Hmmm...I wonder if using `data-no-turbolink` (https://github.com/rails/turbolinks#opting-out-of-turbolinks) on the breadcrumbs that were giving you problems would have helped.  It's a shame because it really made the page loads very fast.